src/options.html

```diff
@@ -67,7 +78,11 @@ <h3>Settings Page</h3>
                 </div>
             </div>
             <div class="row">
+<<<<<<< HEAD
```

Comment:
@stealthanthrax Check some extra lines have to removed that may have come due to the conflict. Please check them :)